### DISCUSSION TITLE
Does detekt respect .editorconfig?

### DISCUSSION THREAD

**Discussion by henrik242** [detekt/detekt#4975] [Q&A] [answered]
I'm in considering migrating from the kotlinter gradle plugin to detekt.  Both projects rely on ktlint for linting, but it seems detekt does not respect the defaults in my `.editorconfig` (such as `max_line_length`).

Here's the start of my `.editorconfig`:
```
root = true

[*.{kts,kt,groovy,java,ts,js,tsx,jsx}]
max_line_length = 140
indent_size = 4
indent_style = space
charset = utf-8
insert_final_newline = true
disabled_rules = no-wildcard-imports,import-ordering
```
I use detekt a gradle plugin in my multiproject, and I get lots of errors such as...

```Line detected, which is longer than the defined maximum line length in the code style. [MaxLineLength]```

**Comment by schalkms** [comment, marked as answer]
I think this is explained pretty nicely in #4962.
If you have further questions, feel free to reach out.

**Comment by henrik242** [comment]
Thanks! Would have been really nice with editorconfig support, but I totally understand if there are no resources or interest among the committers.
